src/net.c

```diff
@@ -443,6 +443,39 @@ static int hyper_set_interface_name(struct rtnl_handle *rth,
 	return 0;
 }
 
+static int hyper_set_interface_mtu(struct rtnl_handle *rth,
```

Comment:
This function duplicates a lot with hyper_set_interface_name(). Can you please make a common version for both of them?